astropy/coordinates/builtin_frames/cirs_observed_transforms.py

```diff
@@ -40,7 +40,8 @@ def cirs_to_altaz(cirs_coo, altaz_frame):
     xp, yp = get_polar_motion(obstime)
 
     #first set up the astrometry context for CIRS<->AltAz
-    astrom = erfa.apio13(obstime.jd1, obstime.jd2,
+    obstime_utc = obstime if obstime.scale == 'utc' else obstime.utc
```

Comment:
I like @taldcroft's suggestion here. The first thing I thought was "I should be able to do ``time.tt.jd1, time.tt.jd2``, because that's the clearest.  But then I checked and realized it's not cached, so I decided it wasn't fast enough given that this is in inner loops.  So I naturally *assumed* there would be a cache and was surprised that there wasn't.

If it helps any, there's precedence in coordinates itself for this.  So I'd advocate for instead doing just as @taldcroft says here.

I think that's a "new feature", though, so it probably has to wait for 1.2 .  In the meantime, I will create a utility function as @mhvk suggests *just for coordinates*, which we can later change to just being ``return getattr(time, scale).jd1, getattr(time, scale).jd2`` or just replace all of it with direct conversions once the caching and such is in.